features/gezag-minderjarige/geen-gezag.feature

```diff
@@ -0,0 +1,31 @@
+#language: nl
+
+Functionaliteit: Als gebruiker wil ik alleen een gezaguitspraak over minderjarige en levende personen
```

Comment:
In principe is `Als gebruiker wil ik alleen een gezaguitspraak over minderjarige en levende personen` geen functionaliteit. Ik zou in dit geval kiezen voor `gezagsbepaling bij meerderjarige of overleden personen`.
Als verduidelijking van de functionaliteit zou dan het volgende worden gebruikt
```
Als domein deskundige/gezag provider
wil ik dat er voor meerderjarige of overleden personen geen gezag wordt geleverd
```

Dit is volgens mij onze keuze en niet die van de API consumers, toch?

Reply:
aangepast